_data/footer-projects.yml

```diff
@@ -0,0 +1,6 @@
+---
+- title: "Waku"
+  link: https://status.im/
```

Comment:
These two links should probably be updated. Assuming things go well with other sites, they should go live at the same time